content/docs/guides/auth/ALTS.md

```diff
@@ -0,0 +1,232 @@
+---
+title: ALTS Authentication
+description: >
+  An overview of gRPC authentication using Application Layer Transport Security
+  (ALTS).
+---
+
+### Overview
+
+Application Layer Transport Security (ALTS) is a mutual authentication and
+transport encryption system developed by Google. It is used for securing RPC
+communications within Google's infrastructure. ALTS is similar to mutual TLS
+but has been designed and optimized to meet the needs of Google's production
+environments. For more information, take a look at the
+[ALTS whitepaper](https://cloud.google.com/security/encryption-in-transit/application-layer-transport-security).
+
+ALTS in gRPC has the following features:
+
+-   Create gRPC servers & clients with ALTS as the transport security protocol.
+-   ALTS connections are end-to-end protected with privacy and integrity.
+-   Applications can access peer information such as the peer service account.
+-   Client authorization and server authorization support.
+-   Minimal code changes to enable ALTS.
+
+gRPC users can configure their applications to use ALTS as a transport security
+protocol with few lines of code. gRPC ALTS is supported in C++, Java, Go, and
+Python.
+
+Note that ALTS is fully functional if the application runs on
+[Google Cloud Platform](https://cloud.google.com/). ALTS could be run on any
+platforms with a pluggable
+[ALTS handshaker service](https://github.com/grpc/grpc/blob/master/src/proto/grpc/gcp/handshaker.proto#L233-L243).
```

Comment:
This should probably reference a specific commit sha instead of just the copy on the master branch.

Thoughts?

Reply:
Good suggestion! Let me update in a separate PR.